docs/gettingstarted/create-landscaper-deployment.md

```diff
@@ -64,7 +65,7 @@ test   test-8qh5w   13m
 The Instance will show the selected ServiceTargetConfig as well as the Installation that has been automatically created by the landscaper service controller.
 
 ```sh
-k get instances.landscaper-service.gardener.cloud test-8qh5w
+k -n laas-user get instances.landscaper-service.gardener.cloud test-8qh5w
```

Comment:
```suggestion
kubectl -n laas-user get instances.landscaper-service.gardener.cloud test-8qh5w
```

Reply:
This was some copy paste error by me. Should be `kubectl` instead of just `k` which probably only works on my machine :-)